arcgis-ios-sdk-samples/Geometry/Format coordinates/FormatCoordinates.storyboard

```diff
@@ -178,16 +47,182 @@
                         <viewLayoutGuide key="safeArea" id="Enn-bV-Rx4"/>
                     </view>
                     <connections>
-                        <outlet property="latLongDDTextField" destination="b1c-Eo-C52" id="19s-nl-iZr"/>
-                        <outlet property="latLongDMSTextField" destination="fra-5S-E0B" id="4Ce-9D-DYm"/>
                         <outlet property="mapView" destination="vLE-Mm-bqf" id="XqS-hA-mtR"/>
-                        <outlet property="usngTextField" destination="H2w-ku-aRr" id="mC3-hc-e7o"/>
-                        <outlet property="utmTextField" destination="dl7-tP-eVK" id="TLI-Ai-X7r"/>
                     </connections>
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="Wlm-1W-Txj" userLabel="First Responder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="713" y="199"/>
+            <point key="canvasLocation" x="712.79999999999995" y="198.35082458770617"/>
+        </scene>
+        <!--Format Coordinates Table View Controller-->
+        <scene sceneID="eIn-QB-OfK">
+            <objects>
+                <tableViewController id="geL-f0-eoD" customClass="FormatCoordinatesTableViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" keyboardDismissMode="onDrag" dataMode="static" style="plain" separatorStyle="default" allowsSelection="NO" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="28" sectionFooterHeight="28" id="5vY-pq-BWx">
```

Comment:
Consider using a grouped table view. Plain table views with static content (ie. fixed number of rows) generally look odd because of the extra blank rows at the bottom.